tests/test_meta/test_grouped_predictor.py

```diff
@@ -13,6 +13,42 @@
 from tests.conftest import general_checks, select_tests
 
 
+@pytest.fixture(name="df_predict_proba")
+def get_df_different_classes_classification():
+
+    np.random.seed(43)
+    group_size = 5
+    n_groups = 2
+
+    group_col = np.repeat(["A", "B"], group_size)
+    x_col = np.random.normal(size=group_size * n_groups)
+    y_col = np.hstack(
+        [
+            np.random.choice([0, 1, 2], size=group_size),
+            np.random.choice([0, 2], size=group_size),
+        ]
+    )
+    df = pd.DataFrame({"group": group_col, "x": x_col, "y": y_col})
+    return df
+
+@pytest.fixture(name="df_predict_proba_request")
+def my_custom_df(request):
```

Comment:
- Following the repo convention you can name the fixture directly instead of using the `name` keyword. 
- If possible I would follow some naming convention as in [conftest.py](https://github.com/koaning/scikit-lego/blob/main/tests/conftest.py). Maybe something along the line of `random_xy_grouped_clf_different_classes` ?(holy moly naming is hard 😂)

Reply:
Yes you're right. 
Changing it to follow the repo's convention. !